ci/Jenkinsfile

```diff
@@ -135,8 +150,18 @@ stage('Deploy PR') {
             unstash 'frontend'
             unstash 'backend'
 
-            echo 'Deploying'
+            echo 'Deploying to PR'
             sh "sudo /usr/local/bin/permaplant-deploy.sh pr ${WORKSPACE}"
+
+            echo 'Loading Data to PR'
+            dir('scraper') {
+                sh 'npm ci'
+                sh 'mkdir ./data/'
+                sh 'cp /nextcloud/mergedDatasets.csv ./data/'
+                sh 'npm run insert'
+                sh 'rm -rf ./data/'
+                sh 'rm -rf ./node_modules/'
```

Comment:
Seems to be unrelated to "Loading Data to PR"? Why was this needed?

Reply:
I just added a manual cleanup, because in elektra Jenkins sometimes leaves old workspace files and those two folders are huge in relation to the very small containers 😸